src/components/schemas/category.yaml

```diff
@@ -75,6 +75,10 @@ category_out_embed:
       description: Represents the name of the sub_category.
     code:
       $ref: ./fields.yaml#/code
+    display_name:
```

Comment:
We agreed to add `display_name` to expenses minimal objects too right ? @kartikeyrajvaidya